### ISSUE TITLE
ewfmount: command not found

### ISSUE THREAD

**Issue by JumpNShootMan** [dfir-scripts/siftgrab#1]
Hello, I'm running siftgrab from Docker.
After I mount my Temp folder from my Windows machine to Docker, I run ermount with a test .e01 file from the terminal.
During the execution of the command, I get the following:

```shell
Use ERMount to mount a disk, disk image
Enter Image File or Device Path: /mnt/temp/
2024-09-13-1558/ sample.E01
Enter Image File or Device Path: /mnt/temp/sample.E01
Image type E01
Source Information
/mnt/temp/sample.E01: EWF/Expert Witness/EnCase image file format

Set Mount Point
Set Path or Enter to Accept Default:
/mnt/image_mount/

Executing ewfmount command: ewfmount /mnt/temp/sample.E01 /mnt/raw
/usr/local/bin/ermount: line 104: ewfmount: command not found
root@c9542829349b:/cases#
``` 

Could you recommend a different approach while using my mounted folder from a Windows machine?
I take it from the example from the SANS website that this process should be working...
I hope this issue can be resolved or that I'm doing something wrong so that I can be corrected.

Thank you for developing this tool, I'll be waiting for a response.

**Comment by JumpNShootMan** [comment]
For more info, this is the command I'm using to run the image:

`docker run -it -v C:/temp:/mnt/ -v C:/temp/media:/media -v C:/temp/cases:/cases dfirscripts/siftgrab`

**Comment by dfir-scripts** [comment]
You can try running siftgrab using the "--privileged" option, however in my testing  the results were not consistent,  The better option is to install ermount outside of the Docker image.  Mount the disk and then, load Siftgrab Docker. The "-v" option in the run command provides a volume mapping outside of the container to your mount points.

**Comment by JumpNShootMan** [comment]
I tried the privileged option, but it didn't work.
Do you think you could help me in terms of the installation of ermount outside of the Docker image? This isn't something I usually do so I'm a bit lost. I apologize if this should be obvious.

**Comment by dfir-scripts** [comment]
If you are running Windows I would recommend installing WSL with Ubuntu 22.04 and then launching the install script from inside Ubuntu WSL.  These commands will install everything you need including ermount.

wget https://raw.githubusercontent.com/dfir-scripts/installers/main/install-forensic-tools.sh
sudo chmod 755 install-forensic-tools.sh
sudo ./install-forensic-tools.sh

**Comment by JumpNShootMan** [comment]
Thank you for your response. I was able to partially solve the issue.
The Ubuntu 22.04 instance was able to install the tools and mount the disk; however...
The Docker instance stopped working every time I tried to run Siftgrab, be it with our without mounted volumes directly from the Windows host.
I'm not sure if you have any more recommendations. I'll try to troubleshoot this issue and if I get it working I'll be sure to add an explanation.

**Comment by JumpNShootMan** [comment]
For extra information, I'm mounting my windows folders to get ermount to read the E01 file.
After that, when I try to run docker with the WSL integration on Ubuntu, the daemon stops and the command docker stops being recognized.
Would appreciate any help.

**Comment by JumpNShootMan** [comment]
I was able to get Siftgrab working by getting rid of the drive mounting process from Windows to the Ubuntu instance.
I manually inserted the E01 files to the Ubuntu instance and then ran docker with the following command:
`sudo docker run -it -v /mnt/:/mnt/ -v /cases/:/cases/ dfirscripts/siftgrab`

Be aware that I added the slash (/) symbols on the paths because otherwise the files would not appear.
A big thanks to the developer for this learning opportunity on WSL and Docker.

**Comment by JumpNShootMan** [comment]
I was able to get Siftgrab working by getting rid of the drive mounting process from Windows to the Ubuntu instance.
I manually inserted the E01 files to the Ubuntu instance and then ran docker with the following command:
`sudo docker run -it -v /mnt/:/mnt/ -v /cases/:/cases/ dfirscripts/siftgrab`

Be aware that I added the slash (/) symbols on the paths because otherwise the files would not appear.
A big thanks to the developer for this learning opportunity on WSL and Docker.
